packages/@aws-cdk/aws-codestarnotifications-targets/test/integ.slack.ts

```diff
@@ -0,0 +1,46 @@
+import * as chatbot from '@aws-cdk/aws-chatbot';
+import * as codebuild from '@aws-cdk/aws-codebuild';
+import * as notifications from '@aws-cdk/aws-codestarnotifications';
+import * as cdk from '@aws-cdk/core';
+import * as targets from '../lib';
+
+class SlackInteg extends cdk.Stack {
+  constructor(scope: cdk.App, id: string, props?: cdk.StackProps) {
+    super(scope, id, props);
+
+    const slack = new chatbot.SlackChannelConfiguration(this, 'MySlackChannel', {
+      slackChannelConfigurationName: 'test-channel',
+      slackWorkspaceId: 'T49239U4W', // modify to your slack workspace id
```

Comment:
If these are not universal, and the deployment will fail for these, then these should not be integ tests. Integ tests should be able to be deployed to anyone's account.

If that is not the case here, please remove this integ tests.

Reply:
Ok, removed this integ tests.